demisto_sdk/__main__.py

```diff
@@ -69,6 +70,22 @@ def __init__(self):
 pass_config = click.make_pass_decorator(DemistoSDK, ensure=True)
 
 
+def check_configuration_file(command, args):
```

Comment:
need to think about the behavior of conflict arguments, for example:
configuration states `-g` and command has the `-i` input.